modules/apps/frontend-js/frontend-js-components-web/src/main/resources/META-INF/resources/forms/input/InputLocalized.tsx

```diff
@@ -47,6 +47,26 @@ const availableLocales = Object.keys(Liferay.Language.available)
 		symbol: language.replace(/_/g, '-').toLowerCase(),
 	}));
 
+const translationsNormalizer = (
+	translations: Liferay.Language.LocalizedValue<string> &
+		Partial<{
+			zh_Hans_CN: string;
+			zh_Hant_TW: string;
+		}>
+) => {
```

Comment:
:thinking: I think that declaring the type of returning value for the `translationsNormalizer` will work

```
const translationNormalizer = (
          translations: Liferay.Language.LocalizedValue<string> &
		Partial<{
			zh_Hans_CN: string;
			zh_Hant_TW: string;
		}>

): Liferay.Language.LocalizedValue<string> { // <= this
 ...
}
```

Reply:
ty @juanjofgliferay ❤️ makes the trick!! 
I think the returning type is inferred :( I´m not understanding the error at all 😓

Reply:
if those `zh_Hans_CN` and `zh_Hant_TW` are valid locale codes, what would make sense to me is to add them to the `Locale` type, so that `Liferay.Language.LocalizedValue` includes them. Do you guys see any reason not to do it?

Reply:
Ouch, I marked it as resolved and muted the thread :( 

As we speak over Slack, avoiding this makes sense to at least try to write the truth with types.

Now the InputLozicer [can receive some extra language keys](https://github.com/liferay-frontend/liferay-portal/commit/d4b0d5d10f4658d4fa295028d21420836a259089#diff-7585cb48d771117115d80891bf883586a6a068c46260c0fbc576c83f96ce2fddR33-R37) and [normalize them](https://github.com/liferay-frontend/liferay-portal/commit/d4b0d5d10f4658d4fa295028d21420836a259089#diff-7585cb48d771117115d80891bf883586a6a068c46260c0fbc576c83f96ce2fddR55); Avoiding spreading the special language keys.